code/game/objects/items/_item_interactions.dm

```diff
@@ -40,3 +40,76 @@
 
 /decl/interaction_handler/storage_open/invoked(atom/target, mob/user, obj/item/prop)
 	target?.storage?.open(user)
+
+/decl/interaction_handler/wash_hands
+	name = "Wash Hands"
+	expected_target_type = /atom
+	interaction_flags = INTERACTION_NEEDS_PHYSICAL_INTERACTION
+
+/decl/interaction_handler/wash_hands/is_possible(atom/target, mob/user, obj/item/prop)
+	. = ..() && target?.reagents?.has_reagent(/decl/material/liquid/water, 150)
+	if(.)
+		for(var/hand_slot in global.all_washable_hand_slots)
+			if(user.get_organ(hand_slot))
+				return TRUE
+
+/decl/interaction_handler/wash_hands/invoked(atom/target, mob/user, obj/item/prop)
+
+	// Probably needs debounce and do_after() but showers and wading into water don't, so whatever.
+	if(!target?.reagents?.has_reagent(/decl/material/liquid/water, 150)) // To avoid washing your hands in beakers.
+		to_chat(user, SPAN_WARNING("\The [src] doesn't have enough water in it to wash your hands."))
+		return
+
+	var/found_hand = FALSE
+	for(var/hand_slot in global.all_washable_hand_slots)
+		if(user.get_organ(hand_slot))
+			found_hand = TRUE
+			break
+	if(!found_hand)
+		return
+
+	var/decl/pronouns/pronouns = user.get_pronouns()
+	user.visible_message(
+		SPAN_NOTICE("\The [user] washes [pronouns.his] hands in \the [target]."),
+		SPAN_NOTICE("You wash your hands in \the [target].")
+	)
+	user.clean()
```

Comment:
wondering if we should have a more specific handwashing proc given that user.clean() is going to wash their entire body. maybe it should just find the covering objects on your hands, wash those, and if there is no covering object, wash your hand organs.
but then again the germ level reduction from washing your hands is in user.clean(), but also germ level stuff needs to get looked at and fixed up on its own...
not blocking, just musing